librosa/core/audio.py

```diff
@@ -521,6 +524,127 @@ def zero_crossings(y, threshold=1e-10, ref_magnitude=None, pad=True,
                   constant_values=pad)
 
 
+@cache
+def clicks(times=None, frames=None, sr=22050, hop_length=512,
+           click_freq=1000.0, click_duration=0.1, click=None, length=None):
+    """Returns a signal with the signal `click` placed at each specified time
+
+    Parameters
+    ----------
+    times : np.ndarray or None
+        times to place clicks, in seconds
+
+    frames : np.ndarray or None
+        frame indices to place clicks
+
+    sr : number
+        desired sampling rate of the output signal
+
+    hop_length : int > 0
+        if positions are specified by `frames`, the number of samples between frames.
+
+    click_freq : float > 0
+        frequency (in Hz) of the default click signal.  Default is 1KHz.
+
+    click_duration : float > 0
+        duration (in seconds) of the default click signal.  Default is 100ms.
+
+    click : np.ndarray or None
+        optional click signal sample to use instead of the default blip.
+
+    length : int > 0
+        desired number of samples in the output signal,
```

Comment:
> I cut that second line because it's not entirely correct now.

You should change the comma to a period.

> Also, I dropped the +1; why was that there?

I don't remember.